rust/stackable-cockpit/src/platform/manifests.rs

```diff
@@ -85,7 +90,7 @@ pub trait InstallManifestsExt {
                     })?;
 
                     let helm_chart: helm::Chart = transfer_client
-                        .get(&helm_file, &Template::new(parameters).then(Yaml::new()))
+                        .get(&helm_file, &Template::new(&parameters).then(Yaml::new()))
```

Comment:
note: If the above clone is removed, this and the borrow below can be removed again.

Reply:
See https://github.com/stackabletech/stackable-cockpit/pull/355/files#r2007550029, not removing the clone